### DISCUSSION TITLE
"Office hours" for the Athens core team

### DISCUSSION THREAD

**Discussion by joelhans** [athensresearch/athens#1348] [Operations]
Hi everyone! We're going to start rolling out weekly office hours for each member of the [core Athens Research team](https://athensresearch.gitbook.io/handbook/company/org-chart/core-team).

Office hours happen in the #coworking channel on our [Discord server](https://discord.gg/as9h8yHNfD). During these hours, one or more core team members will be present and actively working on their area of focus in the Athens universe.

Anyone is welcome to jump in, ask questions, and co-work directly the Athens Research team! Unlike the [community calls](#1282) that happen every Sunday, office hours are designed more specifically for those who want to _contribute_ to Athens and would benefit from working alongside a core team member.

We're still working on dialing in exactly when these calls will happen. Updates to this top comment coming soon.

**Community** with @joelhans and (optionally @mateicanavra): **TBD**
**Engineering** with @neotyk: **TBD**
**Design** with @shanberg: Most weekday evenings, Eastern time.

---

If anyone has seen a community really nail office hours, let me know below. While office hours are meant to be friendly and flexible, it would be nice to attach some qualitative goals to these.

**Comment by joelhans** [comment]
There's been some back-and-forth between the core team about this, and there's some caveats to present for any further discussion:

- Hard for core team members (some are still part-time) to commit to a regular weekly time.
- Regular times might not be amenable to contributors in time zones that don't overlap much with the US.

_Maybe the community would prefer flexible coworking/office hours that change week to week?_

Maybe each core member can promote their office hours for the week using the Apollo bot in our [Discord server](https://discord.gg/as9h8yHNfD).

**Reply by jefftangx** [reply]
Like usual, we can ask: who is this for? Core<>Contributors or Core<>Core?

At what point does it become easier to structure time vs DMing to setting up Discord calls ad-hoc?

**Reply by joelhans** [reply]
This is designed to be for Core<>Contributors. I see it as a lower barrier to entry vs. DMing someone on Discord to set up a 1:1. Some people might not want to do that.

Admittedly, we're not not yet at the scale where office hours are easier than one-off meetings with contributors. I do worry that if we don't establish some kind of system/culture early, we'll perpetually say that we don't have time for this. I've seen that exact outcome before!
